README.md

```diff
@@ -0,0 +1,61 @@
+Installation needed:
```

Comment:
Before the installation notes, mention a few words about this project - what it is about, bla bla.